presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -396,7 +397,10 @@ public TableStatistics getTableStatistics(ConnectorSession session, ConnectorTab
             return EMPTY_STATISTICS;
         }
         List<HivePartition> hivePartitions = partitionManager.getPartitions(metastore, tableHandle, constraint).getPartitions();
-        Map<String, ColumnHandle> tableColumns = getColumnHandles(session, tableHandle);
+        Map<String, ColumnHandle> tableColumns = getColumnHandles(session, tableHandle)
+                .entrySet().stream()
```

Comment:
The best option to avoid such situations would be to add a checkstyle rule which would enforce the proper formatting, then we could avoid such not-the-first-priority comments.